templates/dt_template/README.md

```diff
@@ -0,0 +1,58 @@
+# RHOS Deployed Topology %NEW_DT_NAME%
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/78b3c876eaf9168f9d95b201997ebdc2da42fa02) on Oct 17th, 2023**
+
+## General information
+
+| Revision | Change                | Date             |
+|--------: | :-------------------- | :--------------: |
+| v0.1     | Initial publication   | %TODAY%      |
+
+## Node topology
+| Node role                                     | bm/vm | amount |
+| --------------------------------------------- | ----- | ------ |
+| Openshift master/worker combo-node cluster    | vm    | 3      |
+| Compute nodes                                 | vm    | 3      |
+| Galera DB node, replicating                   | vm    | 3      |
+| Hardware security module                      | bm    | 1      |
+
+## Services, enabled features and configurations
```

Comment:
I would split the table into two: 
- the table like the current one with the list of services which are focus of the deployment and testing of this DT
-  a separate table with just two columns (service and reason) which lists the additional services that needs to be deployed (typically keystone and co) which are not the focus of the testing but they are needed for the full integration testing to happen, under a "### Support services" section

Reply:
Done, good idea